packages/plugin-pnp/sources/PnpLinker.ts

```diff
@@ -142,6 +144,9 @@ export class PnpInstaller implements Installer {
         }
       }
 
+      if (customPackageData.manifest.type === `module`)
+        this.isESMLoaderRequired = true;
```

Comment:
There is, this repo is one of them. While that would work it causes problems with `dlx` as the user has no way to set it, now we could always enable it but I want to avoid the experimental warning as much as possible.